src/main/java/com/minecolonies/api/items/registry/ToolTypeEntry.java

```diff
@@ -0,0 +1,198 @@
+package com.minecolonies.api.items.registry;
+
+import com.minecolonies.api.items.ModToolTypes;
+import com.minecolonies.api.util.constant.IToolType;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+/**
+ * An entry in the ToolType registry that defines the types of
+ * tools within the colony.
+ */
+public class ToolTypeEntry implements IToolType
```

Comment:
I feel this could be kept in same place as old ToolType which was deleted? To at least keep the source break minimal

Reply:
I'm actually not a fan where this class is placed because it has nothing to do with items. Maybe move it to `/api/tools/registry/ToolTypeEntry`, and maybe put some other relevant files there?

Reply:
I would rather keep source break minimal, since this is already binary break, but either dont move or move all

Reply:
I'd probably move all, we have too many "util" files. IToolType can also easily move to the folder I mentioned.

Reply:
Moved to `/api/tools/registry/`